pycytominer/cyto_utils/__init__.py

```diff
@@ -7,6 +7,8 @@
     check_aggregate_operation,
     check_consensus_operation,
     get_pairwise_correlation,
+    check_fields_of_view_format,
+    check_fields_of_view
```

Comment:
make sure to run `black`. I spot a missing comma! :)

Reply:
Done! :)